lib/get-commit.js

```diff
@@ -11,7 +11,7 @@ function getCommit (state) {
 
     .then(function (result) {
       const {filename, patch, blob_url: blobUrl} = result.data.files[0]
-      var branchUrl = blobUrl.replace(/(?:blob)(.+)(?=\/)/g, 'blob/' + state.repoDefaultBranch)
+      var branchUrl = blobUrl.replace(/(?:blob\/)(\w+)/g, 'blob/' + state.repoDefaultBranch)
```

Comment:
I think we I’d change the regex pattern

```diff
- /(?:blob\/)(\w+)/g
+ /\/blob\/[^/]+/
```

That should make it more clear. We don’t need the `g` modifier, either, it’s a single replace. 

The regex will cause problems in edge cases where a repository owner or repository name is "blob", but let’s create a follow up issue for that, it’s out of scope of this PR